cmd/agent/dist/conf.d/docker.d/conf.yaml.default

```diff
@@ -63,10 +63,17 @@ instances:
     # collect_disk_stats: false
 
     ## @param collect_exit_codes - boolean - optional - default: false
-    ## Collect containers exit codes and send service checks critical when exit code is not 0.
+    ## Collect containers exit codes and send service checks critical when exit code is not 0 or 143.
     #
     # collect_exit_codes: false
 
+    ## @param ok_exit_codes - list of integers - optional - default: [0, 143]
+    ## Define the set of exit codes considered OK. It requires the collect_exit_codes parameter to be enabled.
+    #
+    # ok_exit_codes:
+    #   - 0
+    #   - 143
```

Comment:
Since this is specific to Docker, and not general UNIX exit codes, I think including 143 here by default is OK.  And, I suspect anyone who's automated docker knows the number `143` by heart (just like we used to memorize Windows error numbers, right?) it wouldn't hurt to put a comment here as to why this value is included.